src/main/java/controllers/UserProfilePresenter.java

```diff
@@ -1,4 +1,27 @@
 package controllers;
 
+import java.util.HashMap;
+import java.util.ArrayList;
+
+import entities.User;
+import entities.Post;
+import use_cases.UserManager;
+
 public class UserProfilePresenter {
+    public static void main(String[] args) {
+
+        // initialize some empty variables to create a new object User
+        HashMap<String, Integer> like_history = new HashMap<String, Integer>();
+        ArrayList<User> followers = new ArrayList<User>();
+        ArrayList<User> following = new ArrayList<User>();
+        ArrayList<Post> posts = new ArrayList<Post>();
+
+        // create new User current_user
+        User current_user = new User("Glen", "letmein", "Aspriring CS Student",
```

Comment:
I don't think we would run this code in a main method in UserProfilePresenter, wouldn't we write methods like getFollowers which the User Interface class RecipeAppShell calls? So then RecipeAppShell calls getFollowers and gets a list of followers, which it displays in the Terminal

I think we want UserProfilePresenter to have a bunch of methods, but we can discuss in the group chat maybe